docs/integration.rst

```diff
@@ -55,6 +55,24 @@ partition for U-Boot), :ref:`data partition(s) <sec-data-storage>` or similar.
 Since changing the partition layout is hard or even impossible to change in the
 field, make sure it meets both current and possible future requirements.
 
+.. note::
+
+  The ``/etc/fstab`` in your system's root FS (and RAUC's ``system.conf``)
+  should normally use stable paths to refer to the partitions or devices.
+  Especially a filesystem UUIDs (``UUID=<uuid>`` or
+  ``/dev/disk/by-uuid/<uuid>``) should *not* be used, as they are likely to be
+  different after an update.
+
+  Depending on your system design and firmware, good stable paths can be:
+
+  * plain device names: ``/dev/sda``, ``/dev/mmcblk0p1``, ``/dev/nvme0n1p1``, ``/dev/mapper/…``
```

Comment:
Should we mention that these may depend on storage enumeration order in some cases?